Rocket.Chat/Views/Chat/ChatTitleView.swift

```diff
@@ -65,30 +76,19 @@ final class ChatTitleView: UIView {
                 labelLoading?.text = localized("connection.waiting_for_network.banner.message")
             }
         } else {
-            buttonTitle?.isHidden = false
+            isTitleHidden = false
             viewLoading?.isHidden = true
         }
     }
 
     // MARK: IBAction
 
     @IBAction func buttonChannelDidPressed(_ sender: Any) {
-        delegate?.titleViewButtonChannelDidPressed()
+        delegate?.titleViewChannelButtonPressed()
     }
 
-    @IBAction func buttonMoreDidPressed(_ sender: Any) {
-        delegate?.titleViewButtonMoreDidPressed()
+    @IBAction func buttonSearchDidPressed(_ sender: Any) {
```

Comment:
Why do `ChatTitleView` manage the search button? For me it's supposed to be a `UIBarButtonItem`. The title view should limit itself to manage everything that goes between the left and the right bar button items.

Reply:
Thats how the ellipsis button was implemented 😅. I can add it as a `BarButtonItem` if you want.

Reply:
@filipealva done